lib/plugins/aws/remove/lib/stack.js

```diff
@@ -26,8 +27,47 @@ module.exports = {
       .then(() => cfData);
   },
 
+  disassociateUsagePlan() {
```

Comment:
🤔 Haven't had time to take a deep dive into it yet. Will do this next week!

So it's not possible to completely rely on CloudFormation here? Even if `DependsOn` are added to the resources to declare their dependencies explicitly?